### PR TITLE
Set up router rate limiter

### PR DESCRIPTION
**Pull request by timwangmusic**
## Description
Set up router rate limiter with a middleware.

## Solution
* Limit requests per IP per hour to 100. We can make this configurable later.

## Testing
- [ ] Integration testing on Heroku staging
- [ ] Added new unit tests

## Checks
- [x] Have you removed commented code?
- [x] Have you used gofmt to format your code?

### CODE REVIEW

File: planner/planner.go [hunk 1]
```diff
@@ -1014,6 +1018,22 @@ func (p *MyPlanner) GetPlaceDetails(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, place)
 }
 
+func (p *MyPlanner) rateLimiter() gin.HandlerFunc {
+	logger := iowrappers.Logger
+	// 100 requests per hour
+	rate, err := limiter.NewRateFromFormatted("100-H")
```
Comment (sananand007) [inline planner/planner.go:1024]: 100 per hours. ?

**Review by sananand007 — APPROVED** [review]
lg

File: planner/planner.go (hunk 1 above)
Comment (sananand007) [reply planner/planner.go:1024]: Can we test this now ?

Comment (timwangmusic) [reply planner/planner.go:1024]: I tested for lower limit. It works pretty well. Currently it is set to 100 per IP per hour. Should be enough for regular usages.

**Pull request merged**